modules/plugin/imagemosaic/src/test/resources/org/geotools/gce/imagemosaic/test-data/coverage_bands/indexer.properties

```diff
@@ -0,0 +1 @@
+CoverageNameHandlerSPI=org.geotools.gce.imagemosaic.namehandler.FileNameRegexBasedSPI:regex=^([a-zA-Z0-9]*)
```

Comment:
Wondering if this should end with + instead of "*" (I know I know, it's just a test, but people sometimes pick them as examples when documentation is lacking)